src/Storages/StorageMergeTree.cpp

```diff
@@ -765,6 +765,54 @@ void StorageMergeTree::loadMutations()
         increment.value = std::max(increment.value.load(), current_mutations_by_version.rbegin()->first);
```

Comment:
It seems you changed the permissions of the file

Reply:
Thanks.